otter/controller.py

```diff
@@ -91,7 +118,10 @@ def pause_scaling_group(log, transaction_id, scaling_group):
 
     :return: None
     """
-    raise NotImplementedError('Pause is not yet implemented')
+    if not tenant_is_enabled(scaling_group.tenant_id, config_value):
+        raise NotImplementedError("Pause is not yet implemented")
```

Comment:
Probably better to make this say "Pause is not implemented for legacy groups" or something, since "yet" implies it will be